api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -83,6 +83,7 @@ message AccessLogFilter {
     GrpcStatusFilter grpc_status_filter = 10;
 
     // Extension filter.
+    // [#extension-category: envoy.access_loggers.filters]
```

Comment:
I think the category is actually `envoy.access_loggers.extension_filters`, per https://github.com/envoyproxy/envoy/blob/main/source/common/access_log/access_log_impl.h#L272

That name seems a little redundant, but I think we need to match it or else we need to undertake some additional complexity to deprecate the old category and introduce a more succinct one. (We had a deprecation facility for extension names, but not sure about the categories.)

Reply:
OK. I updated all instances to match.